ops/search/extract-hca-projects.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import json
+from urllib.request import urlopen
+from urllib.error import HTTPError
+
+url = "https://service.azul.data.humancellatlas.org/index/projects?catalog=dcp9&size=1000&sort=projectTitle&order=desc"
```

Comment:
It's minor but you don't need all the parameters here
```suggestion
url = "https://service.azul.data.humancellatlas.org/index/projects?catalog=dcp9"
```

Reply:
If you don't specify size, it only gives you 10 results at a time.